spec/ace.md

```diff
@@ -49,6 +51,14 @@ Volumes that are specified in the Pod Manifest are mounted into each of the apps
 For example, say that the worker-backup and reduce-worker both have a `mountPoint` named "work".
 In this case, the executor will bind mount the host's `/opt/tenant1/work` directory into the `path` of each of the matching "work" `mountPoint`s of the two app filesystems.
 
+If the target `path` does not exist in the rendered filesystem, it SHOULD be created, including any missing parent directories. If the target `path` is a non-empty directory, its contents SHOULD be discarded (e.g. obscured by the bind mount). If the target `path` refers to a file, ACE SHOULD remove that file and create a directory in its place.
```

Comment:
Yes, I didn't think of this one. I'd be inclined to just specify that nested volumes are invalid and should be rejected by ACE. Or should mount order be sorted so that volumes can overlap?

This is an issue I'm having right now: I have an ACI to build FreeBSD ports in a completely clean base system (to find dependency issues). I mount `/usr/ports` from base system read-only, and `/usr/ports/distfiles` (downloaded source tarballs) read-write. I mount distfiles as `/usr/ports.distfiles` in the pod and change the location in `/etc/make.conf`, but I'd be in a tough spot if this wasn't configurable.

On the other hand, allowing overlapping ports feels like opening a can of worms, tricky implementation, special cases, and possible vulnerabilities.